### DISCUSSION TITLE
Installed Doris 2.1.3 in CentOS 7, failed to start BE instance, pls help

### DISCUSSION THREAD

**Discussion by tmorning112** [apache/doris#36069] [A - General / Q&A]
I installed Doris 2.1.3 in CentOS 7, failed to start BE instance, the log of be.out is as below:

INFO: java_cmd /opt/amazon-corretto-11.0.19.7.1-linux-x64/bin/java
INFO: jdk_version 11
SLF4J: Class path contains multiple SLF4J bindings.
SLF4J: Found binding in [jar:file:/data1/doris/be/lib/java_extensions/preload-extensions/preload-extensions-jar-with-dependencies.jar!/
org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: Found binding in [jar:file:/data1/doris/be/lib/java_extensions/java-udf/java-udf-jar-with-dependencies.jar!/org/slf4j/impl/Stati
cLoggerBinder.class]
SLF4J: Found binding in [jar:file:/data1/doris/be/lib/hadoop_hdfs/common/lib/slf4j-reload4j-1.7.36.jar!/org/slf4j/impl/StaticLoggerBind
er.class]
SLF4J: See http://www.slf4j.org/codes.html#multiple_bindings for an explanation.
SLF4J: Actual binding is of type [org.slf4j.impl.Reload4jLoggerFactory]
*** Query id: 0-0 ***
*** is nereids: 0 ***
*** tablet id: 0 ***
*** Aborted at 1717813841 (unix time) try "date -d @1717813841" if you are using GNU date ***
*** Current BE git commitID: 2dc65ce356 ***
*** SIGSEGV address not mapped to object (@0x0) received by PID 13721 (TID 13742 OR 0x7fc4221fe700) from PID 0; stack trace: ***
0# doris::signal::(anonymous namespace)::FailureSignalHandler(int, siginfo_t*, void*) at /home/zcp/repo_center/doris_release/doris/be/
src/common/signal_handler.h:421
1# os::Linux::chained_handler(int, siginfo*, void*) in /opt/amazon-corretto-11.0.19.7.1-linux-x64/lib/server/libjvm.so
2# JVM_handle_linux_signal in /opt/amazon-corretto-11.0.19.7.1-linux-x64/lib/server/libjvm.so
3# signalHandler(int, siginfo*, void*) in /opt/amazon-corretto-11.0.19.7.1-linux-x64/lib/server/libjvm.so
4# 0x00007FC45AC28400 in /lib64/libc.so.6
5# je_arena_dalloc_promoted at ../src/arena.c:1277
6# je_free_default at ../src/jemalloc.c:3014
7# __pthread_create_2_1 in /lib64/libpthread.so.0
8# os::create_thread(Thread*, os::ThreadType, unsigned long) in /opt/amazon-corretto-11.0.19.7.1-linux-x64/lib/server/libjvm.so
9# CompilerThread::CompilerThread(CompileQueue*, CompilerCounters*) in /opt/amazon-corretto-11.0.19.7.1-linux-x64/lib/server/libjvm.so
10# CompileBroker::make_thread(_jobject*, CompileQueue*, AbstractCompiler*, Thread*) in /opt/amazon-corretto-11.0.19.7.1-linux-x64/lib/
server/libjvm.so
11# CompileBroker::possibly_add_compiler_threads() in /opt/amazon-corretto-11.0.19.7.1-linux-x64/lib/server/libjvm.so
12# CompileBroker::compiler_thread_loop() in /opt/amazon-corretto-11.0.19.7.1-linux-x64/lib/server/libjvm.so
13# JavaThread::run() in /opt/amazon-corretto-11.0.19.7.1-linux-x64/lib/server/libjvm.so
14# Thread::call_run() in /opt/amazon-corretto-11.0.19.7.1-linux-x64/lib/server/libjvm.so
15# thread_native_entry(Thread*) in /opt/amazon-corretto-11.0.19.7.1-linux-x64/lib/server/libjvm.so
16# start_thread in /lib64/libpthread.so.0
17# clone in /lib64/libc.so.6

Please help to check the reason of failure. Thanks.

**Comment by ixzc** [comment]
please use jdk8

**Comment by tmorning112** [comment]
Thanks for the reply. After I changed the JDK to version 8. The issue got fixed temporarily, but  after the BE instance run a couple of hours within a BE cluster. I got the issue again, I'm not sure the reason is same or not. The new trace is as following. please advise how to fix it. Thanks again.

New Trace from be.out: 

start time: Mon Jun 10 18:31:07 CST 2024
INFO: java_cmd /data1/jdk1.8.0_202//bin/java
INFO: jdk_version 8
SLF4J: Class path contains multiple SLF4J bindings.
SLF4J: Found binding in [jar:file:/data1/doris/be/lib/java_extensions/preload-extensions/preload-extensions-jar-with-dependencies.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: Found binding in [jar:file:/data1/doris/be/lib/java_extensions/java-udf/java-udf-jar-with-dependencies.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: Found binding in [jar:file:/data1/doris/be/lib/hadoop_hdfs/common/lib/slf4j-reload4j-1.7.36.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: See http://www.slf4j.org/codes.html#multiple_bindings for an explanation.
SLF4J: Actual binding is of type [org.slf4j.impl.Reload4jLoggerFactory]
*** Query id: 0-0 ***
*** is nereids: 0 ***
*** tablet id: 317726 ***
*** Aborted at 1718015470 (unix time) try "date -d @1718015470" if you are using GNU date ***
*** Current BE git commitID: 2dc65ce356 ***
*** SIGSEGV address not mapped to object (@0x0) received by PID 31011 (TID 32172 OR 0x7f0d2d75d700) from PID 0; stack trace: ***
 0# doris::signal::(anonymous namespace)::FailureSignalHandler(int, siginfo_t*, void*) at /home/zcp/repo_center/doris_release/doris/be/src/common/signal_handler.h:421
 1# os::Linux::chained_handler(int, siginfo*, void*) in /data1/jdk1.8.0_202/jre/lib/amd64/server/libjvm.so
 2# JVM_handle_linux_signal in /data1/jdk1.8.0_202/jre/lib/amd64/server/libjvm.so
 3# signalHandler(int, siginfo*, void*) in /data1/jdk1.8.0_202/jre/lib/amd64/server/libjvm.so
 4# 0x00007F0F7DA46280 in /lib64/libc.so.6
 5# doris::segment_v2::SegmentWriter::_full_encode_keys[abi:cxx11](std::vector<doris::KeyCoder const*, std::allocator<doris::KeyCoder const*> > const&, std::vector<doris::vectorized::IOlapColumnDataAccessor*, std::allocator<doris::vectorized::IOlapColumnDataAccessor*> > const&, unsigned long, bool) at /home/zcp/repo_center/doris_release/doris/be/src/olap/rowset/segment_v2/segment_writer.cpp:913
 6# doris::segment_v2::SegmentWriter::_generate_short_key_index(std::vector<doris::vectorized::IOlapColumnDataAccessor*, std::allocator<doris::vectorized::IOlapColumnDataAccessor*> >&, unsigned long, std::vector<unsigned long, std::allocator<unsigned long> > const&) at /home/zcp/repo_center/doris_release/doris/be/src/olap/rowset/segment_v2/segment_writer.cpp:1297
 7# doris::segment_v2::SegmentWriter::append_block(doris::vectorized::Block const*, unsigned long, unsigned long) in /data1/doris/be/lib/doris_be
 8# doris::VerticalBetaRowsetWriter::add_columns(doris::vectorized::Block const*, std::vector<unsigned int, std::allocator<unsigned int> > const&, bool, unsigned int) at /home/zcp/repo_center/doris_release/doris/be/src/olap/rowset/vertical_beta_rowset_writer.cpp:85
 9# doris::Merger::vertical_compact_one_group(std::shared_ptr<doris::Tablet>, doris::ReaderType, std::shared_ptr<doris::TabletSchema>, bool, std::vector<unsigned int, std::allocator<unsigned int> > const&, doris::vectorized::RowSourcesBuffer*, std::vector<std::shared_ptr<doris::RowsetReader>, std::allocator<std::shared_ptr<doris::RowsetReader> > > const&, doris::RowsetWriter*, long, doris::Merger::Statistics*, std::vector<unsigned int, std::allocator<unsigned int> >) in /data1/doris/be/lib/doris_be
10# doris::Merger::vertical_merge_rowsets(std::shared_ptr<doris::Tablet>, doris::ReaderType, std::shared_ptr<doris::TabletSchema>, std::vector<std::shared_ptr<doris::RowsetReader>, std::allocator<std::shared_ptr<doris::RowsetReader> > > const&, doris::RowsetWriter*, long, doris::Merger::Statistics*) at /home/zcp/repo_center/doris_release/doris/be/src/olap/merger.cpp:383
11# doris::Compaction::do_compaction_impl(long) at /home/zcp/repo_center/doris_release/doris/be/src/olap/compaction.cpp:371
12# doris::Compaction::do_compaction(long) at /home/zcp/repo_center/doris_release/doris/be/src/olap/compaction.cpp:136
13# doris::CumulativeCompaction::execute_compact_impl() at /home/zcp/repo_center/doris_release/doris/be/src/olap/cumulative_compaction.cpp:79
14# doris::Compaction::execute_compact() at /home/zcp/repo_center/doris_release/doris/be/src/olap/compaction.cpp:118
15# doris::Tablet::execute_compaction(doris::Compaction&) at /home/zcp/repo_center/doris_release/doris/be/src/olap/tablet.cpp:1947
16# std::_Function_handler<void (), doris::StorageEngine::_submit_compaction_task(std::shared_ptr<doris::Tablet>, doris::CompactionType, bool)::$_1>::_M_invoke(std::_Any_data const&) at /var/local/ldb_toolchain/bin/../lib/gcc/x86_64-linux-gnu/11/../../../../include/c++/11/bits/std_function.h:291
17# doris::ThreadPool::dispatch_thread() in /data1/doris/be/lib/doris_be
18# doris::Thread::supervise_thread(void*) at /home/zcp/repo_center/doris_release/doris/be/src/util/thread.cpp:499
19# start_thread in /lib64/libpthread.so.0
20# __clone in /lib64/libc.so.6
